code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1633,6 +1629,18 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 		H.receiving_cpr = FALSE
 		to_chat(src, "<span class='danger'>You need to stay still while performing CPR!</span>")
 
+/mob/living/carbon/human/proc/get_cpr_mod(mob/living/carbon/human/H)
+	if((head && (head.flags_cover & HEADCOVERSMOUTH)) || (wear_mask && (wear_mask.flags_cover & MASKCOVERSMOUTH) && !wear_mask.mask_adjusted))
+		return CPR_CHEST_COMPRESSION_ONLY
+	if((H.head && (H.head.flags_cover & HEADCOVERSMOUTH)) || (H.wear_mask && (H.wear_mask.flags_cover & MASKCOVERSMOUTH) && !H.wear_mask.mask_adjusted))
```

Comment:
At this point, I'd spin this into a helper function called "get_chest_compression" or something similar since it is used twice and contain multiple conditional

Reply:
fixed in the latest commit.